addons/mail/models/mail_render_mixin.py

```diff
@@ -271,6 +271,7 @@ def _render_eval_context(self):
             'is_html_empty': is_html_empty,
             'slug': self.env['ir.http']._slug,
             'user': self.env.user,
+            'env': self.env,
```

Comment:
... huh? :eyes: 

cc @tde-banana-odoo

Reply:
What does it have to do with using `properties.xxx_id.name` though? It's not super clear.

Reply:
So it never worked then? :thinking: 

In any case I'm not super comfortable merging this change, would like the opinion of the mail master himself :smile: 

Maybe "it's fine because you could already access the env", but could it have further implications?

Reply:
@awa-odoo It worked in normal qweb template, but not in "inline template"